1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,11 @@
 */
 
 function evenNumbers(n) {
```

Comment:
This implementation does not produce a comma-separated string of numbers - it just prints the numbers :) 

You are almost there - you need a small step to make this work